repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapper.scala

```diff
@@ -129,13 +130,13 @@ class HttpServletRequestWrapper(originalRequest: HttpServletRequest, val inputSt
 
   def getSplittableHeaderScala(headerName: String): List[String] =
     getHeadersScala(headerName).foldLeft(List.empty[String])((list, s) => list ++ s.split(","))
-      .map(_.trim)
+      .map(_.trim).filter(StringUtils.isNotBlank(_))
 
   def getHeadersScala(headerName: String): List[String] = {
     if (removedHeaders.contains(headerName)) {
       List.empty[String]
     } else {
-      headerMap.getOrElse(headerName, super.getHeaders(headerName).asScala.toList)
+      headerMap.getOrElse(headerName, super.getHeaders(headerName).asScala.toList).filter(StringUtils.isNotBlank(_))
```

Comment:
I believe you should be able to replace:
`.filter(StringUtils.isNotBlank(_))`
with:
`.filter(StringUtils.isNotBlank)`

Which cuts out a few unnecessary characters.

Reply:
Updated.